plugins/uefi-capsule/fu-uefi-device.h

```diff
@@ -84,6 +84,12 @@ typedef enum {
  * Do not prepend a plausible missing capsule header.
  */
 #define FU_UEFI_DEVICE_FLAG_NO_CAPSULE_HEADER_FIXUP (1 << 7)
+/**
+ * FU_UEFI_DEVICE_FLAG_ENABLE_DEBUGGING:
+ *
+ * Increase debugging level for the device.
+ */
+#define FU_UEFI_DEVICE_FLAG_ENABLE_DEBUGGING (1 << 8)
```

Comment:
`FU_UEFI_DEVICE_FLAG_ENABLE_EFI_DEBUGGING` perhaps

Reply:
Yup, that's better.